main.go

```diff
@@ -289,6 +292,79 @@ Arguments:
 	verifyCmd.Flags().StringVar(&serverConfigPathArg, "config-path", "/etc/opk/config.yml", "Path to the server config file. Default: /etc/opk/config.yml.")
 	rootCmd.AddCommand(verifyCmd)
 
+	clientCmd := &cobra.Command{
+		Use:     "client [subcommand]",
+		Short:   "Interact with client configuration",
+		Example: `  opkssh client provider list`,
+		Args:    cobra.ExactArgs(0),
+	}
+
+	providerCmd := &cobra.Command{
+		Use:     "provider [subcommand]",
+		Short:   "Interact with provider configuration",
+		Example: `  opkssh provider list`,
```

Comment:
Shouldn't this example be `opkssh client provider list` or am i confused about how this works?

Reply:
fixed